tests/test_utils.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+
+import os
+import sys
+
+from mirdata import utils
+
+import json
+import pytest
+
+if sys.version_info.major == 3:
+    builtin_module_name = 'builtins'
+else:
+    builtin_module_name = '__builtin__'
```

Comment:
@drubinstein maybe a better way to do this?

Reply:
`from future import builtins`